build-image/Dockerfile

```diff
@@ -4,6 +4,10 @@
 # default when running `docker buildx build` or when DOCKER_BUILDKIT=1 is set
 # in environment variables.
 
+# NOTE: The GO_RUNTIME is used to switch between the default google go runtime and  mcr.microsoft.com/oss/go/microsoft/golang:1.22-bullseye which is a microsoft
+# fork of go that allows using windows crypto instead of boring crypto. Details at https://github.com/microsoft/go/tree/microsoft/main/eng/doc/fips
+ARG GO_RUNTIME=golang:1.22.0-bullseye
+
```

Comment:
nit: should we remove the default so we're forced to update it in the Jsonnet consistently?